src/game.js

```diff
@@ -209,6 +218,14 @@
             // reset the frame counter
             frameCounter = 0;
             frameRate = ~~(0.5 + 60 / me.sys.fps);
+            
+            // set step size based on the updatesPerSecond
+            stepSize = (1000 / Math.min(me.sys.updatesPerSecond, me.sys.fps));
```

Comment:
Got it, that's the case I didn't consider.

What happens if `update` always takes longer than 16.6667ms? Do we get an infinite regress of slower updates? There's no way it could possibly catch up...